src/openmrs-react-root-decorator.js

```diff
@@ -69,3 +80,16 @@ export default function decorateOptions(userOpts) {
     };
   };
 }
+
+function I18nextLoadNamespace(props) {
```

Comment:
This component does two things:

1. Updates our react applications whenever the i18next language changes.
2. Downloads the translation strings for the current ES module